app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -33,7 +33,8 @@ module Api::V2::TaxonomiesController
       param :ignore_types, Array, N_("List of resources types that will be automatically associated"), :required => false
       resource_name = (param_name == :location) ? 'organization' : 'location'
       resource_ids = "#{resource_name}_ids".to_sym
-      param resource_ids, Array, N_("Associated %{resource} IDs") % { resource: _(resource_name) }, :required => false
+      # TODO: translate resource_name?
```

Comment:
I'm dumb for writing `_(resource_name)` instead of simply translating taxonomy names. Although, due to some language specialties, this would seems odd in the end anyway.

In this case, I'd better be more explicit or WET:

```ruby
# ...
mapping = { location: ['organization', N_("Associated organization IDs")], organization: ['location', N_("Associated location IDs")] }
resource_ids = "#{mapping[param_name]&.at(0)}_ids".to_sym # & is just to be sure it doesn't crash, but can be omitted I guess
# ...
param resource_ids, Array, mapping[param_name]&.at(1), :required => false # & is for the same reason, can be changed to [1]
```